src/Main.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import request from 'superagent';
+import PokeList from './PokeList';
+import Spinner from './Spinner';
+
+export default class Main extends Component {
+
+
+    state = {
+        pokedex: [],
+        loading: false,
+        query: '',
+        direction: 'asc',
+        type: '',
+    }
+
+    componentDidMount = async () => {
+        await this.fetchPokeData();
+    }
+
+    fetchPokeData = async () => {
+        this.setState({ loading: true });
+
+        const URL = this.state.query ? `https://pokedex-alchemy.herokuapp.com/api/pokedex?pokemon=${this.state.query}&sort=pokemon&direction=${this.state.direction}` : `https://pokedex-alchemy.herokuapp.com/api/pokedex?sort=pokemon&direction=${this.state.direction}`
+
+        const data = await request.get(URL);
+        console.log(data);
+        this.setState({ loading: false })
+        this.setState({ pokedex: data.body.results })
+    }
+
+    fetchTypeData = async () => {
```

Comment:
Sofia! Great job making an entirely new fetch to solve this and a button! Definitely seems really clean and works well!